openwb.conf

```diff
@@ -225,6 +225,12 @@ soci3intervall1=10
 soccarnetintervall=10
 carnetuser='user'
 carnetpass='pass'
+soc_vag_username='user'
```

Comment:
Neu Einstellungen bitte nicht in der `openwb.conf` eintragen. Bei einem Update wird die durch eine lokale Sicherung wieder ersetzt. Hierfür ist die Datei `runs/updateConfig.sh` gedacht.

Reply:
Danke für den Hinweis